packages/hooks/src/useCFDAccounts.ts

```diff
@@ -0,0 +1,18 @@
+import { useStore } from '@deriv/stores';
+
+const useCFDAccounts = () => {
+    const { client } = useStore();
+    const { dxtrade_accounts_list, mt5_login_list } = client;
+
+    let cfd_accounts: typeof mt5_login_list = [];
+    if (Array.isArray(mt5_login_list)) {
+        cfd_accounts = [...cfd_accounts, ...mt5_login_list];
```

Comment:
[Question]: Why do you need to spread `cfd_accounts`?

Reply:
@matin-deriv I also didn't understand this part and I just copied/pasted the code from traders-hub-store but as I checked with @thisyahlen-deriv , we are covering all kinds of cfd accounts  in this part. 
maybe later I can find a better way to get all cfd accounts :thinking: